seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/operation/sink/SinkRegisterOperation.java

```diff
@@ -56,7 +56,7 @@ public void run() throws Exception {
         SinkAggregatedCommitterTask<?> task = null;
         for (int i = 0; i < RETRY_TIME; i++) {
             try {
-                task = server.getTaskExecutionService().getExecutionContext(committerTaskID.getTaskGroupID())
```

Comment:
Can the encapsulation of the group-id be improved? Hide details in TaskExecutionService, TaskID is unique for a job, task-group-id can be hidden.

Reply:
Our current TaskID is not unique and may be repeated in different jobs or pipelines

Reply:
I will create an issue for this as an improvement